stackslib/src/net/download/nakamoto/tenure_downloader_set.rs

```diff
@@ -472,15 +571,23 @@ impl NakamotoTenureDownloaderSet {
                     &naddr, &downloader.tenure_id_consensus_hash
                 );
                 finished.push(naddr.clone());
-                finished_tenures.push(downloader.tenure_id_consensus_hash.clone());
+                finished_tenures.push(CompletedTenure::from(downloader));
                 continue;
             }
             debug!(
                 "Send request to {} for tenure {} (state {})",
                 &naddr, &downloader.tenure_id_consensus_hash, &downloader.state
             );
             let Ok(sent) = downloader.send_next_download_request(network, neighbor_rpc) else {
-                debug!("Downloader for {} failed; this peer is dead", &naddr);
```

Comment:
There's three new info logs here, how chatty do you expect them to be? How often do these log lines appear in a normally running node?

Reply:
Seconded. Prob greatly depends on the deprioritization timeout , no? as once marked deprioritized, we shouldn't hit this branch for a specific naddr much.

Reply:
> There's three new info logs here, how chatty do you expect them to be? How often do these log lines appear in a normally running node?

Not that often. I've been testing this with `info!`-only logs, and the deprioritization timeout does a good job at limiting the frequency of this log entry.